pom.xml

```diff
@@ -234,6 +238,18 @@
                 </reportSets>
             </plugin>
 
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <reportSets>
+                    <reportSet>
+                        <reports>
+                            <!-- select non-aggregate reports -->
```

Comment:
I have to admit that I have no idea what "select non-aggregate reports" means in this context 🙈 if you can think of a simpler explanation, you're welcome to add it, else I'd happily offer to give you a suggestion if you explain to me what it means.